apps/dsky_clock/metadata.json

```diff
@@ -0,0 +1,17 @@
+{ "id": "dsky_clock",
+    "name": "DSKY Clock",
+    "shortName":"dsky_clock",
+    "icon": "dsky_clock.img",
```

Comment:
I'd prefer if the icon name was changed to `app.png` to keep to the standard naming of the repo. I believe it has to be a png at least.

Reply:
I was following the icon section here:
https://github.com/espruino/EspruinoDocs/blob/master/tutorials/Bangle.js%20First%20App.md

when I was testing, if I just put the PNG then the icon doesn't seem to show up in my menu.  Happy to rename and/or upload the png version if needed tho?

Sorry, first time noob questions.

Reply:
Yes the js-coded icon is used for the watch (seems to be the one you have, I misinterpreted. So it should be renamed to `app-icon.js` in the repo) but the png is used for the app loader. 

More here: https://www.espruino.com/Bangle.js+App+Loader

Reading that now I found it a bit hard to follow - maybe some steps have been swapped around. But look and see if it helps! :)